Feature/hdf 97 update main css stylesheet

tracyarciaga opened the pull request:
<!-- You can erase any parts of this template not applicable to your Pull Request. -->
## Description of Changes
<!-- Enter description of changes here. -->
- Update colors and fonts in theme.
- Change CSS for BalancerPool,  PoolSummary and Header components.
- Add new images.

[Link to Jira Ticket -  HDF 97](https://halodao.atlassian.net/browse/HDF-97?atlOrigin=eyJpIjoiYzQwZGJhY2NmNDE5NDdmNDkzMTcyMzlhN2I4N2U1NjEiLCJwIjoiaiJ9)
[Link to Jira Ticket  -  HDF 98](https://halodao.atlassian.net/browse/HDF-98)
[Link to Jira Ticket  -  HDF 99](https://halodao.atlassian.net/browse/HDF-99)
[Link to Jira Ticket  -  HDF 100](https://halodao.atlassian.net/browse/HDF-100)

### Developer Checklist:

* [x] I have followed the guidelines in our Contributing document
* [x] This PR has a corresponding JIRA ticket
* [x] My branch conforms with our naming convention i.e. `feature/HDF-XXX-description`
* [x] All files have appropriate file headers and documentation
* [x] I have written new tests for your core changes, as applicable
* [ ] I have successfully ran tests locally

### Reviewers Checklist:
* [ ] Code is readable and understandable; any unclear parts have explanations 
* [ ] UI/UX changes match the corresponding figma/other design resources, if applicable
* [ ] I have successfully ran tests locally

schystz commented:
Hi @tracyarciaga , can you please rebase this branch to develop so there are less files/changes to review on Files changed tab? I believe that should also solve the code conflicts shown above.

tracyarciaga commented:
@schystz rebase done.

schystz commented:
> @schystz rebase done.

Looks like rebase failed @tracyarciaga  -

<img width="890" alt="Screen Shot 2021-04-12 at 3 36 55 PM" src="https://user-images.githubusercontent.com/1032707/114357966-f0b0ea00-9ba4-11eb-9243-e031e9f1c166.png">

tracyarciaga commented:
> > @schystz rebase done.
> 
> Looks like rebase failed @tracyarciaga -
> 
> <img alt="Screen Shot 2021-04-12 at 3 36 55 PM" width="890" src="https://user-images.githubusercontent.com/1032707/114357966-f0b0ea00-9ba4-11eb-9243-e031e9f1c166.png">

hmmm
.. I don't understand why, also there are no conflicts already.  The file 
changed tab contains only the actual files that have been changed too.
@schystz

schystz commented:
Yeah it looks like this is because of the previous merge in `develop` - the commits where directly pushed to develop branch for some reason instead of the branch (`features/hdf-8-view-stats-per-pool`) merged to `develop`. But it looks like the PR is all good now, I can continue to review. Thanks @tracyarciaga ! :)

tracyarciaga commented:
> Yeah it looks like this is because of the previous merge in `develop` - the commits where directly pushed to develop branch for some reason instead of the branch (`features/hdf-8-view-stats-per-pool`) merged to `develop`. But it looks like the PR is all good now, I can continue to review. Thanks @tracyarciaga ! :)

Thank you!  I also fixed the ESlint warnings.

schystz commented on public/locales/en.json:
```diff
@@ -87,5 +87,10 @@
+  "stake": "Stake",
+  "poolSummaryStakeable": "My Stakeable Value",
+  "poolSummaryStaked": "My Staked Value",
+  "poolSummaryHaloEarned": "My HALO Earned"
```
Nice catch @tracyarciaga , I forgot this codebase uses localization so we should take advantage of it too

tracyarciaga replied:
@schystz thanks!

schystz commented on src/pages/BalancerPool/index.tsx:
```diff
@@ -32,33 +61,58 @@ const BalancerPool = () => {
+              </TYPE.darkGray>
+            </Row>
+            <Row>
+              <StyledExternalLink href="https://google.com" style={{ fontSize: '14px' }}>
```
@tracyarciaga maybe we link this to `#` for now until it is available, but definitely avoid google.com :P

schystz commented on src/pages/BalancerPool/index.tsx:
```diff
@@ -32,33 +61,58 @@ const BalancerPool = () => {
+            </TitleRow>
+            <Row>
+              <TYPE.darkGray style={{ marginRight: '0.5rem', marginTop: '0.5rem', justifySelf: 'flex-start' }}>
+                Stake your Balancer Pool Tokens (BPT) to earn Halo!
```
@tracyarciaga noticed this line is not directly the same as figma design

schystz reviewed (commented):
@tracyarciaga code changes looks good to me in general aside from a few minor comments.

Just pasting here some UI issues Chris and I has observed:

<img width="484" alt="Screen Shot 2021-04-12 at 3 58 16 PM" src="https://user-images.githubusercontent.com/1032707/114363600-20fb8700-9bab-11eb-93e3-1a3e59fa8e44.png">

<img width="635" alt="Screen Shot 2021-04-12 at 3 41 23 PM" src="https://user-images.githubusercontent.com/1032707/114363622-28229500-9bab-11eb-9f09-e965157dff94.png">

<img width="518" alt="Screen Shot 2021-04-12 at 3 41 43 PM" src="https://user-images.githubusercontent.com/1032707/114363633-2ce74900-9bab-11eb-9256-dc2c401586c7.png">

tracyarciaga commented:
@schystz Pushed update to resolve your comments. Thank you!

ernestpascual commented:
Hi guys, i raised a PR that might affect this PR https://github.com/HaloDAO/halodao-interface/pull/14 
FYI, thanks!

tracyarciaga commented:
@schystz Pushed the changes to address your comments:
1.  Added test -> "(Liquidity Pool)" in  the Farm header.
2. Reverted colors for network, balance, address items in navbar.

schystz reviewed (approved):
Looks good on my side, so approving!

ernestpascual commented:
hi @tracyarciaga  one minor comment:

<img width="1091" alt="Screen Shot 2021-04-15 at 1 44 51 AM" src="https://user-images.githubusercontent.com/10893681/114755506-4abfd400-9d8c-11eb-83e8-c65023ae1b7f.png">

I saw this already fix in @schystz previous PR. We can keep it to 2 decimals. Otherwise, im good!

ernestpascual commented:
on the vesting side i think I have the fix for that, will just merge with this branch after this is merged to develop :)

schystz commented:
> hi @tracyarciaga one minor comment:
> 
> <img alt="Screen Shot 2021-04-15 at 1 44 51 AM" width="1091" src="https://user-images.githubusercontent.com/10893681/114755506-4abfd400-9d8c-11eb-83e8-c65023ae1b7f.png">
> 
> I saw this already fix in @schystz previous PR. We can keep it to 2 decimals. Otherwise, im good!

Actually for this one, we'll be tracking it in a separate task - https://halodao.atlassian.net/browse/HDF-142

ernestpascual commented:
gotcha @schystz , approved!

tracyarciaga reviewed (approved):
Approving this. Tested in my local.
FE integration changes with the new rewards contract are working in Farm and Vesting.

ernestpascual reviewed (approved):
everything seems to be working well. approving!
